tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "target": "es6",
+    "lib": ["dom", "dom.iterable", "esnext", "es6"],
```

Comment:
I'm sure this is necessary, but could you explain? (this is mostly for my understanding).

Reply:
I remember I had a reason for doing this, but I just tried it with es5 and everything seems fine. Must've been something I tried that didn't work for something else.

Reply:
I'd expect you did too, so maybe it'll come back to bite us. Leaving this conversation unresolved just in case we run into issues later on.